<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1223</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>221</width>
      <height>751</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_academic_informations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Academic Informations</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_semester">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Semester</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_edit_informations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit Infomations</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_course">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add Coursse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_settings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>670</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tab_main">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>1001</width>
      <height>751</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>991</width>
        <height>711</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_information">
       <attribute name="title">
        <string>Basic Informations</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>GPA</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>50</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hours Passed</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>270</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>49</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>490</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Warnings</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>490</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Level</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>380</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>160</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Grade</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Courses</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>135</y>
          <width>981</width>
          <height>531</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Code</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Course Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Grade</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>GPA</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Degree</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Credit Hours</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Semester</string>
         </property>
        </column>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>30</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abs</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>80</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abs</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>30</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Grade</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>80</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Semester</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Code</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>161</width>
          <height>36</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Credit Hours</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_3">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>80</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>A-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Abs</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>161</width>
          <height>36</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>80</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_semester">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="table_add_semester">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>981</width>
        <height>451</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>160</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>58</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Title</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Credit Hours</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Degree</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Instructor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prerequisites</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_save">
      <property name="geometry">
       <rect>
        <x>368</x>
        <y>667</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_table_row">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Row</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_delete_table_row">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete Row</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_semester_number">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>71</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>Semester Number</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_instructor">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>160</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Instructor</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>575</x>
        <y>71</y>
        <width>106</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>Spring</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fall</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Summer</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Semester Type</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Course</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit Informations</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
